notifications/templates.ts

```diff
@@ -0,0 +1,45 @@
+import { MessageType } from './messageTypes';
+import { GraphQLUser } from '../common/user/session';
+import { Concrete_notification as ConcreteNotification } from '../graphql/generated';
+
+// only actionPath or url is optionally allowed but not both
+type NotificationAction = { navigateTo?: string; url?: undefined } | { url: string; navigateTo?: undefined };
+export type MessageTemplate = {
```

Comment:
Maybe it makes sense to already distinguish "MessageTemplate" and "Message" here?

Reply:
I have renamed it to NotificationMessage now, I don't see a place for a second type